src/components/dialogs/two-windings-transformer/create-rule-dialog.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    Grid,
+} from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { gridItem } from '../dialogUtils';
+import { useInputForm, useTextValue } from '../inputs/input-hooks';
+import {
+    PHASE_TAP,
+    RATIO_TAP,
+} from './two-windings-transformer-creation-dialog';
+
+export const CreateRuleDialog = (props) => {
+    const inputForm = useInputForm();
+
+    const [lowTapValue, lowTapValueField] = useTextValue({
+        ...(props.ruleType === PHASE_TAP && { label: 'LowTapAlpha' }),
+        ...(props.ruleType === RATIO_TAP && { label: 'LowTapRatio' }),
+        validation: { isFieldRequired: true },
+        inputForm: inputForm,
+    });
+
+    const [highTapValue, highTapValueField] = useTextValue({
+        ...(props.ruleType === PHASE_TAP && { label: 'HighTapAlpha' }),
+        ...(props.ruleType === RATIO_TAP && { label: 'HighTapRatio' }),
+        validation: { isFieldRequired: true },
+        inputForm: inputForm,
+    });
+
+    const isTapValuesValid = () => {
+        if (highTapValue && lowTapValue) {
+            return !(highTapValue - lowTapValue > 0);
```

Comment:
I used useTextValue because useDoubleValue caused an issue which doesn't seem to apply anymore, thus I replaced useTextValue with useDoubleValue to prevent string values